blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionHandler.java

```diff
@@ -61,13 +64,32 @@ public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
 
         Tracker.getInstance().track(request);
 
+        requestCount.inc();
+
+        if ( !mediaTypeChecker.isContentTypeValid(request.headers()) ) {
```

Comment:
I am kinda hesitant to add more methods in DefaultHandler. That class really needs to be re-written. It does not need to have all static methods like that.